doc-content/enterprise-only/vehicle-route-planning/vrp-run-openshift.adoc

```diff
@@ -0,0 +1,56 @@
+[id='vrp-run-openshift_{context}']
+
+= Run OptaWeb Vehicle Routing on {OPENSHIFT}
+
+Linux users can use the `runOnOpenShift.sh` Bash script to install OptaWeb Vehicle Routing on {OPENSHIFT}.
+
+NOTE:  The `runOnOpenShift.sh` script does not run on macOS.
+
+//You can use Red Hat CodeReady Containers to set up a single-node OpenShift 4 cluster on your local computer. CodeReady Containers provides a minimal preconfigured OpenShift cluster for development and testing purposes. It is delivered as a Red Hat Enterprise Linux virtual machine.
+
+.Prerequisites
+
+* You have access to an OpenShift cluster and the OpenShift command-line interface (`oc`) has been installed. For information about {OpenShift}, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html-single/installing/index[_Installing OpenShift Container Platform_].
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* Internet access is available.
+
+.Procedure
+. Log in to or start a {OPENSHIFT} cluster.
+
+////
+.. Add the OpenShift command-line interface (`oc`) to your `$PATH`:
++
+[source,shell]
+----
+eval $(crc oc-env)
+----
+
+.. Log in as the `developer` user:
++
+[source,shell]
+----
+oc login -u developer -p developer https://api.crc.testing:6443
+----
+////
+.. Enter the following command where `<PROJECT_NAME>` is the name of your new project:
++
+[source]
+----
+oc new-project <PROJECT_NAME>
+----
+.. If necessary, change directory to the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaweb-vehicle-routing/sources` directory.
+
+.. Enter the following command to execute the `runOnOpenShift.sh` script:
++
+[source]
+----
+./runOnOpenShift.sh <OSM_FILE_NAME> <COUNTRY_CODE_LIST> <OSM_FILE_DOWNLOAD_URL>
+----
++
+In this command, replace the following variables:
+
+* `<OSM_FILE_NAME>`: An OpenStreetMap (OSM) file that you downloaded from `http://download.geofabrik.de`
+* `<COUNTRY_CODE_LIST>`: A comma-separated list of country codes  For a list of country codes, see https://www.iso.org/iso-3166-country-codes.html[_ISO 3166 Country Codes_].
+* `<OSM_FILE_DOWNLOAD_URL>`: The URL for the OSM file download
++
+NOTE: For help with the `runOnOpenShift.sh` script, enter `/runOnOpenShift.sh --help`.
```

Comment:
`/runOnOpenShift.sh --help` needs to be replaced with `./runOnOpenShift.sh --help` bcs shell script lanches with '.'

Reply:
:-) I think you are the engineer :-) Changed!